RecoEgamma/EgammaElectronProducers/plugins/GsfElectronCoreEcalDrivenProducer.cc

```diff
@@ -1,79 +1,102 @@
-
-#include "GsfElectronCoreEcalDrivenProducer.h"
-
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
-
 #include "DataFormats/EgammaCandidates/interface/GsfElectronCore.h"
 #include "DataFormats/ParticleFlowReco/interface/GsfPFRecTrack.h"
 #include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
-#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
```

Comment:
The include of `ElectronSeedFwd.h` is removed because this is already correctly included in `ElectronSeed.h`. The SuperCluster include is removed because again it's a `Fwd` and we only use the `reco::SuperClusterRef` as an argument for one of the member functions in `GsfElectronCore` [1]. So again I think it's fair to get this definition indirectly via `GsfElectronCore.h`.

[1] https://github.com/cms-sw/cmssw/blob/master/DataFormats/EgammaCandidates/interface/GsfElectronCore.h#L55